Resources/public/css/theme/views/fieldedit.css

```diff
@@ -48,25 +34,22 @@
             transform: rotate(-45deg);
 }
 
-.ez-editfield-infos .ez-tail-up-tooltip:before {
-    margin-top:-5px;
-    margin-left: 4px;
-    border-top: 1px solid #ccc;
-    border-right: 1px solid #ccc;
+.is-using-touch-device .ez-editfield-input-area .ez-tail-down-tooltip .ez-fielddescription-tooltip-text {
+    margin-left: 1.5em;
 }
 
-.ez-editfield-infos .ez-tail-up-tooltip .ez-fielddefinition-tooltip-text {
-    margin-top: -8px;
+.ez-editfield-input-area .ez-tail-down-tooltip .ez-fielddescription-tooltip-text {
+    margin-top: 0;
 }
 
-.ez-editfield-infos .ez-tail-down-tooltip:after {
+.ez-editfield-input-area .ez-tail-down-tooltip:after {
     margin-top:-9px;
-    margin-left: 4px;
+    margin-left: 4em;
     border-left: 1px solid #ccc;
     border-bottom: 1px solid #ccc;
 }
 
-.ez-editfield-infos .ez-fielddefinition-name {
```

Comment:
this removes the bold style on the field definition name when editing a content, was that intended ?